client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/ContactsSection/ContactsTabs.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {Divider, Tab, Tabs, Collapse, useTheme} from '@material-ui/core';
+import {GroupOutlined} from '@material-ui/icons';
+import { faUserEdit } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import ManualContactsForm from './ManualContactsForm/ManualContactsForm';
+import useFormStyles from 'styles/formStyles';
+import useStyles from './ContactsTabsStyles';
+
+enum contactFormTabs {
+    manualContactsForm = 0,
+    familyMembers = 1,
+}
+
+const FormComponent = ({currentTab}: {currentTab: number;}) => {
+    const classes = useStyles();
+
+    return <>
+        <Collapse classes={{container: classes.collapse}}
+                  in={currentTab === contactFormTabs.manualContactsForm}>
+            <ManualContactsForm/>
+        </Collapse>
+    </>
+};
+
+const ContactsTabs = ({isVisible}: {isVisible: boolean}) => {
+    const [currentTab, setTab] = React.useState<contactFormTabs>(0);
+    const formClasses = useFormStyles();
+    const classes = useStyles();
+    const {palette: {action: {active}}} = useTheme();
+    const tabClasses = {wrapper: classes.tab, selected: classes.selected, labelIcon: classes.icon};
+
+    const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
+        setTab(newValue);
+    };
+
+    return (
+        <div className={isVisible ? classes.tabs : formClasses.hidden}>
+            <Tabs classes={{indicator: classes.tabIndicator}}
+                indicatorColor='primary'
+                orientation='vertical'
+                value={currentTab}
+                onChange={handleChange}
+            >
+                <Tab classes={tabClasses}
```

Comment:
and maybe with the mentioned array you can do the same here

Reply:
icons would need to be made into seperate components since they recieve hooks props... 
also i think it would ne less readable